spec/glossary.adoc

```diff
@@ -2,12 +2,19 @@
 = Glossary
 
 [glossary]
+[[active_rule]]Active Rule:: An active rule is an _instance_ of a <<rule,rule>> that is enabled for a given <<sonar_project>>, with possibly different metadata than the rule. For example severity and parameters can be overridden for active rules, so that different <<sonar_project,Sonar Projects>> can be analyzed for the same rule, but with different behavior/threshold.
 [[binding]]Binding:: In <<connected_mode>>, this is the association between a <<configuration_scope,configuration scope>> and a <<sonar_project,Sonar project>>, for a given <<connection,connection>>.
 [[binding_clue]]Binding Clue:: For a <<configuration_scope,configuration scope>>, this is any element that could help to guess what should be the <<binding,binding>> in <<connected_mode,connected mode>>. A configuration scope can have multiple binding clues, partial, or even contradictory. They will be processed during xref:binding_suggestion.adoc#binding_suggestion[binding suggestion computation].
 [[configuration_scope]]Configuration Scope:: Represent an abstract IDE element that can be configured, and possibly <<binding,bound>>. Usually a project (Eclipse, IntelliJ), module (IntelliJ), solution (VS, Rider), or workspace folder (VSCode).
 [[connected_mode]]Connected Mode:: The use of SonarLint associated with a SonarQube server or SonarCloud. See also <<standalone_mode,standalone mode>>.
 [[connection]]Connection:: A channel of communication between SonarLint and a particular SonarQube server or SonarCloud. See also <<connected_mode,connected mode>>.
 [[connection_config]]Connection Configuration:: All attributes that will be used to instantiate and operate a <<connection,connection>>. Usually provided by the IDE settings. It includes credentials, endpoint URL, timeouts, proxy...
+[[finding]]Finding:: Generic concept that includes all things reported by <<sonar_analyzer,Sonar analyzers>>: issues, security hotspots, taint vulnerabilities, ...
 [[project_key]]Project Key:: Unique identifier of a <<sonar_project,Sonar project>> in a given SonarQube or SonarCloud instance. Project key can be changed on server side (the server use uuid internally).
+[[rule]]Rule:: A rule is something Sonar products can assess on the user's code. Each <<sonar_analyzer,Sonar analyzer>> will contribute a set of rules. Rules are identified by a <<rule_key,rule key>>. Rules have attached metadata (name, type, tags, description, default activation...). Rules can also have parameters, with default values. On SonarQube or SonarCloud, users can create quality profiles, that are logical rule sets in which they can decide to activate some rules, and override some rules metadata (severity, parameters, ...). <<sonar_project,Sonar projects>> are associated to a quality profile. When analyzing the code of a project, <<sensor,sensors>> are only interested in enabled rules and the effective parameter values. This is what we call <<active_rule,active rules>>.
```

Comment:
```suggestion
[[rule]]Rule:: A rule is a set of criteria that Sonar products can assess on the user's code. Each <<sonar_analyzer,Sonar analyzer>> will contribute a set of rules. Rules are identified by a <<rule_key,rule key>>. Rules have attached metadata (name, type, tags, description, default activation...). Rules can also have parameters, with default values. On SonarQube or SonarCloud, users can create quality profiles, that are logical rule sets in which they can decide to activate some rules, and override some rules metadata (severity, parameters, ...). <<sonar_project,Sonar projects>> are associated with a quality profile. When analyzing the code of a project, <<sensor,sensors>> are only interested in enabled rules and effective parameter values. This is what we call <<active_rule,active rules>>.
```

Reply:
Is really 

> a set of criteria

clearer than

> something

? :smile: 

@damien-urruty-sonarsource any suggestion?